funcx_endpoint/funcx_endpoint/cli.py

```diff
@@ -0,0 +1,226 @@
+    # execute
+    state = CommandState.ensure()
+    setup_logging(debug=state.debug)
+
+
+@app.command("version")
+@common_options
+def version_command():
+    """Show the version of funcx-endpoint"""
+    import funcx_endpoint
+
+    click.echo(f"FuncX endpoint version: {funcx_endpoint.__version__}")
+
+
+@app.command(name="configure", help="Configure an endpoint")
+@click.option("--endpoint-config", default=None, help="a template config to use")
+@name_option
+@common_options
+def configure_endpoint(*, name: str, endpoint_config: str | None):
+    """Configure an endpoint
+
+    Drops a config.py template into the funcx configs directory.
+    The template usually goes to ~/.funcx/<ENDPOINT_NAME>/config.py
+    """
+    endpoint = get_cli_endpoint()
+    endpoint.configure_endpoint(name, endpoint_config)
+
+
+@app.command(name="start", help="Start an endpoint by name")
+@name_option
+@click.option(
+    "--endpoint-uuid", default=None, help="The UUID for the endpoint to register with"
+)
+@common_options
+def start_endpoint(*, name: str, endpoint_uuid: str | None):
+    """Start an endpoint
+
+    This function will do:
+    1. Connect to the broker service, and register itself
+    2. Get connection info from broker service
+    3. Start the interchange as a daemon
+
+
+    |                      Broker service       |
+    |               -----2----> Forwarder       |
+    |    /register <-----3----+   ^             |
+    +-----^-----------------------+-------------+
+          |     |                 |
+          1     4                 6
+          |     v                 |
+    +-----+-----+-----+           v
+    |      Start      |---5---> EndpointInterchange
+    |     Endpoint    |         daemon
+    +-----------------+
+    """
+    endpoint = get_cli_endpoint()
+    endpoint_dir = os.path.join(endpoint.funcx_dir, name)
+
+    if not os.path.exists(endpoint_dir):
+        configure_command = "funcx-endpoint configure"
+        if name != "default":
+            configure_command += f" -n {name}"
+        msg = (
+            f"\nEndpoint {name} is not configured!\n"
+            "1. Please create a configuration template with:\n"
+            f"\t{configure_command}\n"
+            "2. Update the configuration\n"
+            "3. Start the endpoint\n"
+        )
+        click.echo(msg)
+        return
+
+    try:
+        endpoint_config = SourceFileLoader(
+            "config", os.path.join(endpoint_dir, endpoint.funcx_config_file_name)
+        ).load_module()
+    except Exception:
+        log.exception(
+            "funcX v0.2.0 made several non-backwards compatible changes to the config. "
+            "Your config might be out of date. "
+            "Refer to "
+            "https://funcx.readthedocs.io/en/latest/endpoints.html#configuring-funcx"
+        )
+        raise
+    endpoint.start_endpoint(name, endpoint_uuid, endpoint_config)
+
+
+@app.command("stop")
+@name_option
```

Comment:
Same comment as the one on top, having to specify `funcx-endpoint stop --name <EP_NAME>` is not ideal.

Reply:
As above, this was reverted.